finch.yaml

```
@@ -148,27 +148,28 @@ provision:
 - mode: user
   script: |
     #!/bin/bash
+
+    # Enable SSHing into the VM as root (e.g., in `nerdctlConfigApplier.Apply`).
+    sudo cp ~/.ssh/authorized_keys /root/.ssh/
     sudo chown $USER /mnt/lima-finch
 
-    # https://github.com/containerd/nerdctl/blob/cffdf87ff4d648a5344eea1406bb95ca3ad7eaa4/extras/rootless/containerd-rootless.sh#L144-L146
-    # XDG_DATA_HOME & ~/.local/share: https://github.com/containerd/nerdctl/blob/cffdf87ff4d648a5344eea1406bb95ca3ad7eaa4/extras/rootless/containerd-rootless.sh#L51
-    mkdir -p /mnt/lima-finch/containerd ~/.local/share/containerd
-    sudo mount --bind /mnt/lima-finch/containerd ~/.local/share/containerd
+   # https://github.com/containerd/containerd/blob/main/docs/ops.md#base-configuration
```

Missing one space at the beginning of this line
```suggestion
    # https://github.com/containerd/containerd/blob/main/docs/ops.md#base-configuration
```

---

Updated.

---

> Interesting, I guess my tests weren't as in depth as yours. I think we can actually fix that by mounting the persistent data volume to both the old rootless and new rootfull locations. Can you try that out?

This actually works. However should we make this change though just to support rootless containers which were created prior to this change?